src/Stryker.Core/Stryker.Core/Options/Inputs/ReportFileNameInput.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Abstractions;
+using Stryker.Core.Exceptions;
+
+namespace Stryker.Core.Options.Inputs
+{
+    public class ReportFileNameInput : Input<string>
+    {
+        protected override string Description => string.Empty;
+
+        public override string Default => null;
+
+        public string Validate()
+        {
+            if (string.IsNullOrWhiteSpace(SuppliedInput))
+            {
+                return "mutation-report";
+            }
+            if (SuppliedInput.IndexOfAny(Path.GetInvalidPathChars()) != -1)
```

Comment:
Maybe we should use [`Path.GetInvalidFileNameChars()`](https://docs.microsoft.com/en-us/dotnet/api/system.io.path.getinvalidfilenamechars?view=net-5.0) as we validate a filename and not a filepath!